api/proto/config/netconfig.proto

```diff
@@ -69,4 +70,7 @@ message WifiConfig {
     }
     cryptoblock crypto = 20;         // encrypted block
     int32  priority = 25;            // priority of connection, default is 0
-}
\ No newline at end of file
+
+    // contains the cipher payload
```

Comment:
Fine, but you should 1) mark identity and password above with comments that they will be deprecated and replaced by the X and Y inside the CipherBlock, and 2) remove the above cryptoblock.

Reply:
Sure. Will do.